nix/craneCommon.nix

```diff
@@ -72,6 +72,7 @@ craneLib.overrideScope' (self: prev: {
     PROTOC = "${pkgs.protobuf}/bin/protoc";
     PROTOC_INCLUDE = "${pkgs.protobuf}/include";
     CARGO_PROFILE = self.commonProfile;
+    ALLOW_GIT_FAIL = "true";
```

Comment:
A convention is to prefix your env variable name with a name of the project they belong to. `CARGO_`, `PROTOC_`, etc. For us it would be `FEDIMINT_`. Maybe `FEDIMINT_BUILD_` even?